src/main/java/spireMapOverhaul/zones/brokenSpace/relics/BrokenSneckoEye.java

```diff
@@ -0,0 +1,34 @@
+package spireMapOverhaul.zones.brokenSpace.relics;
+
+import basemod.helpers.CardModifierManager;
+import com.megacrit.cardcrawl.actions.common.MakeTempCardInHandAction;
+import com.megacrit.cardcrawl.cards.AbstractCard;
+import com.megacrit.cardcrawl.relics.SneckoEye;
+import spireMapOverhaul.zones.brokenSpace.cardmods.UnreadableCardMod;
+
+import static spireMapOverhaul.util.Wiz.adp;
+
+public class BrokenSneckoEye extends BrokenRelic {
+    public static final String ID = "BrokenSneckoEye";
+    public static final int AMOUNT = 1;
+
+    public BrokenSneckoEye() {
+        super(ID, RelicTier.SPECIAL, LandingSound.CLINK, SneckoEye.ID);
+
+    }
+
+    @Override
+    public void atTurnStart() {
+        for (int i = 0; i < AMOUNT; i++) {
+            AbstractCard c = adp().masterDeck.getRandomCard(true).makeStatEquivalentCopy();
```

Comment:
The new effect for this is much better (and the unreadable mod no longer having constantly changing text is much more pleasant). One unreadable card per turn that you can decide whether you want to risk playing seems fine.

One thing that needs to be changed about this -- sources of card generation like this need to avoid healing cards (anything where `card.hasTag(CardTags.HEALING)` returns true). You should add in logic to prevent those cards from being generated.